basiclibrary/src/main/java/basiclibrary/WeatherAnalyzer.java

```diff
@@ -0,0 +1,77 @@
+package basiclibrary;
+
+import java.util.*;
+
+public class WeatherAnalyzer {
+
+    public static void main(String[] args) {
+        // within your main method...
+        List<String> votes = new ArrayList<>();
+        votes.add("Bush");
+        votes.add("Bush");
+        votes.add("Bush");
+        votes.add("Shrub");
+        votes.add("Hedge");
+        votes.add("Shrub");
+        votes.add("Bush");
+        votes.add("Hedge");
+        votes.add("Bush");
+
+        String winner = tally(votes);
+        System.out.println(winner + " received the most votes!");
+    }
+    public String analyzeWeather(int[][] arr){
+        HashSet<Integer> weatherDataSet = new HashSet<>();
+
+        int min = 999, max = 0;
```

Comment:
When hard coding a value, it is standard to write the variable name in all caps:
int MIN = 999, MAX = 0;

If you are hard coding a value and the variable has more than one word then separate them with a _:
int MIN_VALUE = 999, MAX_VALUE = 0

Also, it's very common to see developers initialize variables like this with the first value in whatever data structure they are working with (what you did is totally fine!):
int MIN = arr[0][0], MAX = [0][0]